crates/sui-types/src/object.rs

```diff
@@ -611,3 +611,19 @@ impl Default for ObjectFormatOptions {
         }
     }
 }
+
+impl Display for ObjectRead {
```

Comment:
Thanks @lxfind for this.  I think if you derive Debug it will pretty much print out the same thing.... except for Exists